### DISCUSSION TITLE
Is there a way to change the output variable name for a particular key?

### DISCUSSION THREAD

**Discussion by alobaili** [rogerluan/arkana#34] [Q&A] [answered]
I have a secret in the `.arkana.yml` named `APIKey` and the generated Swift variable name for it is `aPIKey`.

Is there a way to configure Arkana so that the generated Swift variable name is `apiKey`?

**Comment by rogerluan** [comment, marked as answer]
Hi @alobaili 👋 

Unfortunately, not at the moment. This could be implemented, but this wasn't planned in the roadmap.

One thing you could do is be more specific and name the service, e.g `StripeAPIKey` → `stripeAPIKey`. In addition, this would make it more clear to readers what the API key is about.

Would that help for now?

**Reply by alobaili** [reply]
Yes I can rename the key. Thanks
